### PR TITLE
Fixed linter issues and enabled travis

### LINKED ISSUE #21: set up Travis and add linter checks

### PR DESCRIPTION
**Pull request by 0xsarvesh**
fixes #21

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,18 +27,18 @@
...
+    "chai": "4.2.0",
+    "eslint": "5.16.0",
+    "eslint-config-airbnb-base": "13.1.0",
+    "eslint-plugin-import": "2.17.3",
+    "ethlint": "1.2.4",
+    "ganache-cli": "^6.7.0",
+    "mocha": "6.2.0",
+    "openzeppelin-solidity": "2.1.1",
+    "solium": "1.2.4",
+    "truffle": "5.0.0-beta.2",
```
Comment (deepesh-kn) [inline package.json]: We can move out of beta here, for truffle latest release is now`5.1.0`.
Similarly, for `web3` we can move out of beta version, latest is `1.2.4`

**Review by deepesh-kn — CHANGES_REQUESTED** [review]
Just a minor change: 

In .travis.yml
Change
```
script:
  - npm run compile
  - npm run test:contracts
```
to 
```
script:
  - npm run compile
  - npm run test
```

File: package.json [hunk 2]
```diff
@@ -27,18 +27,18 @@
...
-    "solium": "^1.2.4",
-    "truffle": "^5.0.0-beta.2",
+    "assert": "2.0.0",
+    "bn.js": "5.0.0",
+    "chai": "4.2.0",
+    "eslint": "5.16.0",
+    "eslint-config-airbnb-base": "13.1.0",
+    "eslint-plugin-import": "2.17.3",
+    "ethlint": "1.2.4",
+    "ganache-cli": "^6.7.0",
```
Comment (abhayks1) [inline package.json]: 🤔 
```suggestion
    "ganache-cli": "6.7.0",
```

**Review by benjaminbollen — APPROVED** [review]
LGTM, has a conflict in package-lock.json to be resolved

**Comment by 0xsarvesh** [comment]
> LGTM, has a conflict in package-lock.json to be resolved

Done

File: package.json (hunk 1 above)
Comment (0xsarvesh) [reply package.json]: Updated the truffle version. But web3 version 1.2.4 is causing a test failure. Needed to investigate separately. Created ticket #29

**Review by deepesh-kn — APPROVED** [review]

**Pull request merged**